ci/js_token_swap_script.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e
+
+(cd ../../token/js && npm install)
```

Comment:
Let's not assume a PWD, that'll break when a user runs it.

Also more kabab case - `ci/js-token-swap-script.sh`.   The `-script` suffix seems unnecessary too